javascript/javascript3/week2/main.js

```diff
@@ -0,0 +1,46 @@
+const redBoxElement = document.querySelector('ul.marks li:nth-child(1)');
+const blueBoxElement = document.querySelector('ul.marks li:nth-child(2)');
+const greenBoxElement = document.querySelector('ul.marks li:nth-child(3)');
+//Should translate the circles one at a time from their random start position
+function translateOneByOne() {
+    const p1 = new Promise(resolve => {
+        setTimeout(() => {
+            resolve(moveElement(redBoxElement, { x: 20, y: 300 }))
+        }, 1000)
+    })
```

Comment:
No need to wrap `moveElement` with a promise. It is a function that returns a promise. So your variable `p1` can be simply like this
```js
const p1  = moveElement(redBoxElement, { x: 20, y: 300 })
```
and the same for the rest